cfgov/jinja2/v1/_includes/organisms/content-banner.html

```diff
@@ -0,0 +1,33 @@
+{# ==========================================================================
+
+   Banner
+
+   ==========================================================================
+
+   Description:
+
+   Create an banner when given:
+
+    value:         				An object used to customize the markup.
+
+   	value.border_top:       	Whether banner should have a top border.
+
+   	value.border_bottom:    	Whether banner should have a bottom border.
+
+   	value.content:          	Banner content
+
+   	value.additional_classes: 	Additional banner classes
+
+   ========================================================================== #}
+	<div class="content_banner 
```

Comment:
ok, so I would just move it out of the organism folder and remove the references in top docs.